development.html

```diff
@@ -54,34 +55,27 @@ <h1 style="font-size: 17px;font-family: 'Inter', sans-serif;font-weight: normal;
 
       <div class="col">
 
-        <img src="img/młynmaria.jpeg" style="margin-top: 60px;">
+        <img class="photo_młyn_małe" src="img/młynmaria.jpeg">
 
     </div>
   </div>
 </div>
 
-<section class="project" style="background-color: white; margin-top: -40px;">
+<section class="project background_light_with_margins"></section>
 
-</section>
-
-   <section class="project" style="background-color: #FFFAF0; padding: 0px;">
+<center><h1 style="font-size: 30px;font-family: 'Josefin Sans', sans-serif;font-weight: normal; text-align: center;;"><b>realizacja projektu</b></h1></center>
```

Comment:
https://www.w3schools.com/tags/tag_center.asp

Center łatwo ogarnąć flexem albo text-align: center. Zostały tutaj jeszcze inlinowe style :)

Reply:
jak mu daje tylko text-align:center to leci na lewą stronę (whyyy), na tę chwilę zostawię center, ale włożę go w tagi